androdeb

```diff
@@ -66,6 +67,12 @@ case $key in
 esac
 done
 
+if [ ! -z "$KERNELSRC" -a ! -z "KBUILDOUT" ];
+then
+   c_warning "--kernelsrc option will set --kbuildout option ineffective"
```

Comment:
why? I think you need both the kernelsrc and kernelout paths. kernelsrc also has headers that need to be extracted. Overall your patch seems to be incomplete.

Reply:
When building kernel out of tree, "source" directory will be linked to the original kernel source directory.
kernel_source/Makefile
  489 ifneq ($(KBUILD_SRC),)
  490     $(Q)ln -fsn $(srctree) source
  491     $(Q)$(CONFIG_SHELL) $(srctree)/scripts/mkmakefile \
  492         $(srctree) $(objtree) $(VERSION) $(PATCHLEVEL)
  493 endif